src/c_api/c_api.cc

```diff
@@ -188,13 +188,29 @@ int MXNDArrayCreate(const mx_uint *shape,
   API_END();
 }
 
+int MXNDArrayCreateExInt64(const mx_int64 *shape,
+                           mx_uint ndim,
+                           int dev_type,
+                           int dev_id,
+                           int delay_alloc,
+                           int dtype,
+                           NDArrayHandle *out) {
+  API_BEGIN();
+  *out = new NDArray(
+      mxnet::TShape(shape, shape + ndim),
+      Context::Create(static_cast<Context::DeviceType>(dev_type), dev_id),
+      delay_alloc != 0,
+      dtype);
+  API_END();
+}
+
 int MXNDArrayCreateEx(const mx_uint *shape,
-                    mx_uint ndim,
-                    int dev_type,
-                    int dev_id,
-                    int delay_alloc,
-                    int dtype,
-                    NDArrayHandle *out) {
+                      mx_uint ndim,
```

Comment:
int?

Reply:
keping it consistent with previous implementation

Reply:
Can you please paste the tests that this may break?

Reply:
it breaks build itself